orchestration/hca_manage/validation.py

```diff
@@ -4,31 +4,37 @@
 from dagster_utils.contrib.google import parse_gs_path
 from google.cloud.storage import Client
 from hca.staging_area_validator import StagingAreaValidator
-
 from hca_manage.common import DefaultHelpParser
 
 
 class HcaValidator:
-    def validate_staging_area(self, path: str, ignore_inputs: bool, client: Client) -> Any:
+    def validate_staging_area(
+        # self, path: str, retries: int, ignore_inputs: bool, client: Client
+        self,
+        path: str,
+        ignore_inputs: bool,
+        client: Client,
+    ) -> Any:
         """
         Run the validation pre-checks on the staging area
         :param path: Google Cloud Storage path for staging area
         """
         adapter = StagingAreaValidator(
             staging_area=path,
+            # total_retries=retries,
```

Comment:
Can this commented out parameter be removed?

Reply:
I'm leaving it for now as I'd like to revisit this feature on my next go around with this codebase